plugins/modules/aci_bfd_interface_policy.py

```diff
@@ -0,0 +1,311 @@
+       }
+   raw:
+     description: The raw output returned by the APIC REST API (xml or json)
+     returned: parse error
+     type: str
+     sample: '<?xml version="1.0" encoding="UTF-8"?><imdata totalCount="1"><error code="122" text="unknown managed object class "/></imdata>'
+   sent:
+     description: The actual/minimal configuration pushed to the APIC
+     returned: info
+     type: list
+     sample:
+       {
+           "fvTenant": {
+               "attributes": {
+                   "descr": "Production environment"
+               }
+           }
+       }
+   previous:
+     description: The original configuration from the APIC before the module has started
+     returned: info
+     type: list
+     sample:
+       [
+           {
+               "fvTenant": {
+                   "attributes": {
+                       "descr": "Production",
+                       "dn": "uni/tn-production",
+                       "name": "production",
+                       "nameAlias": "",
+                       "ownerKey": "",
+                       "ownerTag": ""
+                   }
+               }
+           }
+       ]
+   proposed:
+     description: The assembled configuration from the user-provided parameters
+     returned: info
+     type: dict
+     sample:
+       {
+           "fvTenant": {
+               "attributes": {
+                   "descr": "Production environment",
+                   "name": "production"
+               }
+           }
+       }
+   filter_string:
+     description: The filter string used for the request
+     returned: failure or debug
+     type: str
+     sample: ?rsp-prop-include=config-only
+   method:
+     description: The HTTP method used for the request to the APIC
+     returned: failure or debug
+     type: str
+     sample: POST
+   response:
+     description: The HTTP response from the APIC
+     returned: failure or debug
+     type: str
+     sample: OK (30 bytes)
+   status:
+     description: The HTTP status from the APIC
+     returned: failure or debug
+     type: int
+     sample: 200
+   url:
+     description: The HTTP url used for the request to the APIC
+     returned: failure or debug
+     type: str
+     sample: https://10.11.12.13/api/mo/uni/tn-production.json
+   """
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec, aci_annotation_spec
+
+
+def main():
+    argument_spec = aci_argument_spec()
+    argument_spec.update(aci_annotation_spec())
+    argument_spec.update(
+        name=dict(type="str", aliases=["bfd_interface_policy"]),
+        description=dict(type="str"),
+        admin_state=dict(type="str", default="enabled", choices=["enabled", "disabled"]),
+        detection_multiplier=dict(type="int", default=3),
+        min_transmit_interval=dict(type="int", default=50),
+        min_receive_interval=dict(type="int", default=50),
```

Comment:
Do we require defaults set, or does apic default when not provided? If apic defaults, remove the default and just send None.

Reply:
Also adjust description to state apic defaults to when not provided in that case

Reply:
changed.